integr8ly/integreatly-operator
# Add an automated test for SKU

# Description
This is one of PRs for [MGDAPI-1488] (https://issues.redhat.com/browse/MGDAPI-1488)
In this test, SKU configuration (are we setting correct values in CRs) is checked along with the SKU change event and ensuring that it is possible to increase resources outside of the SKU change event and they will not be set back. 

## Verification steps 
1. ensure you have access to a cluster with RHOAM installed from a feature branch: log in to the cluster.
2. run the following command in your terminal (NOTE: ensure you have reconciler running during the test) 
```
INSTALLATION_TYPE=managed-api TEST=A34 make test/e2e/single 
```
The test will run approximately for 13 minutes. 
Here are some of the commands that would allow you to track progress along with the console output: 
```
watch -n5 "oc get rhmi rhoam -n redhat-rhoam-operator -o yaml | yq - r 'status.stage'"
watch -n5 "oc get rhmi rhoam -n redhat-rhoam-operator -o yaml | yq - r 'status.sku'"  
watch -n5 "oc get rhmi rhoam -n redhat-rhoam-operator -o yaml | yq - r 'status.toSKU'"
```

## Type of change


## Checklist
